migrations/versions/d9ba1b15d196_.py

```diff
@@ -0,0 +1,105 @@
+"""
+Rename ontology_ids and add external_id's to missing entities
+
+Revision ID: d9ba1b15d196
+Revises: 83788dfcbd00
+Create Date: 2018-04-25 16:26:19.227853
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'd9ba1b15d196'
+down_revision = '83788dfcbd00'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('biospecimen', sa.Column('ncit_id_anatomical_site', sa.Text(), nullable=True))
+    op.add_column('biospecimen', sa.Column('ncit_id_tissue_type', sa.Text(), nullable=True))
+    op.add_column('biospecimen', sa.Column('source_text_anatomical_site', sa.Text(), nullable=True))
+    op.add_column('biospecimen', sa.Column('source_text_tissue_type', sa.Text(), nullable=True))
+    op.add_column('biospecimen', sa.Column('source_text_tumor_descriptor', sa.Text(), nullable=True))
+    op.add_column('biospecimen', sa.Column('spatial_descriptor', sa.Text(), nullable=True))
+    op.add_column('biospecimen', sa.Column('uberon_id_anatomical_site', sa.Text(), nullable=True))
+    op.drop_column('biospecimen', 'tissue_type')
+    op.drop_column('biospecimen', 'uberon_id')
+    op.drop_column('biospecimen', 'tumor_descriptor')
+    op.drop_column('biospecimen', 'anatomical_site')
+    op.add_column('diagnosis', sa.Column('icd_id_diagnosis', sa.Text(), nullable=True))
+    op.add_column('diagnosis', sa.Column('mondo_id_diagnosis', sa.Text(), nullable=True))
+    op.add_column('diagnosis', sa.Column('ncit_id_diagnosis', sa.Text(), nullable=True))
+    op.add_column('diagnosis', sa.Column('source_text_diagnosis', sa.Text(), nullable=True))
+    op.add_column('diagnosis', sa.Column('source_text_tumor_location', sa.Text(), nullable=True))
+    op.add_column('diagnosis', sa.Column('spatial_descriptor', sa.Text(), nullable=True))
+    op.add_column('diagnosis', sa.Column('uberon_id_tumor_location', sa.Text(), nullable=True))
+    op.drop_column('diagnosis', 'tumor_location')
+    op.drop_column('diagnosis', 'diagnosis')
```

Comment:
For the future, we will want to use `alter` instead of `drop_column` to preserve the data.